src/reaper_osara.cpp

```diff
@@ -1723,16 +1744,19 @@ void reportTracksWithState(const char* prefix, TrackStateCheck checkState) {
 	ostringstream s;
 	s << prefix << ": ";
 	int count = 0;
-	for (int i = 0; i < CountTracks(0); ++i) {
-		MediaTrack* track = GetTrack(0, i);
+	// We use -1 for the master track.
+	for (int i = -1; i < CountTracks(0); ++i) {
+		MediaTrack* track;
+		if (i == -1)
+			// Even when the master track is invisible, it could have a state we want to know, such as muted.
+			track = GetMasterTrack(0);
```

Comment:
I realise my older code doesn't do this, but I'm shifting newer code to always brace the bodies of `if`, `else`, `while`, etc. blocks, even if they contain only one statement. This prevents confusion and potentially unintentional brokenness; e.g. if someone added an extra line without realising they needed to brace the block. This particular block is a prime example of the  confusion risk: the comment doesn't count as code, so we don't need braces here, even though there are actually multiple lines in this block because of the comment.